scripts-base/src/componentTypes.ts

```diff
@@ -2,17 +2,15 @@ export type ComponentEl = HTMLElement | SVGElement | Window
 
 export type EventMapByElement<E> = E extends Window
 	? WindowEventMap
-	: E extends SVGElement
-	? SVGElementEventMap
-	: E extends HTMLBodyElement
-	? HTMLBodyElementEventMap
-	: E extends HTMLVideoElement
-	? HTMLVideoElementEventMap
-	: E extends HTMLAudioElement
-	? HTMLMediaElementEventMap
-	: E extends HTMLElement
-	? HTMLElementEventMap
-	: never
+	: (E extends SVGElement
+			? SVGElementEventMap
+			: (E extends HTMLBodyElement
+					? HTMLBodyElementEventMap
+					: (E extends HTMLVideoElement
+							? HTMLVideoElementEventMap
+							: (E extends HTMLAudioElement
+									? HTMLMediaElementEventMap
+									: (E extends HTMLElement ? HTMLElementEventMap : never)))))
```

Comment:
👍

Reply:
Je důvod, proč to nezformátovat jako

```ts
export type EventMapByElement<E> =
	E extends Window ? WindowEventMap :
	E extends SVGElement ? SVGElementEventMap :
	E extends HTMLBodyElement ? HTMLBodyElementEventMap :
	...
```
?

Reply:
@JanTvrdik Ano, Prettier. :neutral_face:. Takhle přesně jsem to měl původně.